finagle-http/src/main/scala/com/twitter/finagle/http/codec/HttpDtab.scala

```diff
@@ -44,17 +44,30 @@ object HttpDtab {
   private def pathFailure(path: String, cause: IllegalArgumentException) =
     Failure("Invalid path: "+path, cause)
 
+  private def prefixFailure(prefix: String, cause: IllegalArgumentException) =
+    Failure("Invalid prefix: "+prefix, cause)
+
   private def nameFailure(name: String, cause: IllegalArgumentException) =
     Failure("Invalid name: "+name, cause)
 
   private def decodePath(b64path: String): Try[Path] =
```

Comment:
Oh, oh, sorry.  I thought you were referring to prefixFailure -- I'll remove decodePath